core/src/main/scala-2.13/org/apache/pekko/persistence/postgres/db/ExtendedPostgresProfile.scala

```diff
@@ -0,0 +1,46 @@
+package org.apache.pekko.persistence.postgres.db
+
+import com.github.tminglei.slickpg.*
+import slick.basic.Capability
+import slick.jdbc.JdbcCapabilities
+
+trait ExtendedPostg2resProfile
```

Comment:
type: wild `2`

Reply:
Yeah, took me more then I'm willing to admit to catch this one 😓